src/pages/publication/SubmissionPanel.tsx

```diff
@@ -36,9 +36,18 @@ interface SubmissionPanelProps {
 
 const SubmissionPanel: React.FC<SubmissionPanelProps> = ({ savePublication }) => {
   const { t } = useTranslation('publication');
-  const { errors, values, setFieldValue }: FormikProps<FormikPublication> = useFormikContext();
+  const { errors, setFieldTouched, setFieldValue, values }: FormikProps<FormikPublication> = useFormikContext();
   const history = useHistory();
 
+  const setAllFieldsTouched = useCallback(() => {
```

Comment:
trenger ikke egen funksjon her heller _hvis_ det bare skal kalles på mount?

Reply:
det er du som har lagt til dette, så jeg antok at du gjorde det for en grunn? 😅jeg har ikke endret på noe av logikken